.github/workflows/ci_tests.yaml

```diff
@@ -7,6 +7,12 @@ on:
   push:
     branches: [ master ]
   pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - '*.md'
+      - '*.json'
+      - 'README.rst'
+      - 'LICENSE.txt'
```

Comment:
So if a git commit updates any file that matches these patterns (`doc/**`, `*.md`, etc), that means the workflow won't run? I guess that's ok, as long as we don't commit changes to a *.py and *.md file at the same time, or make sure to trigger the test somehow by editing only a *.py file.

 A problem is that someone with administrative privileges might be required to merge the commit if the 'required checks' are not triggered. But I guess that's ok.

Reply:
> So if a git commit updates any file that matches these patterns (`doc/**`, `*.md`, etc), that means the workflow won't run?

No. This is what the documentation says:

> When using the push and pull_request events, you can configure a workflow to run when at least one file does not match paths-ignore or at least one modified file matches the configured paths.

If we make changes to a *.py and a .md file, the workflow still runs.

Reply:
> If we make changes to a *.py and a .md file, the workflow still runs.

Ah ok, perfect!